### ISSUE TITLE
Finetune Swin model for object detection on custom dataset without masking labels

### ISSUE THREAD

**Issue by PatelKrutarth** [NielsRogge/Transformers-Tutorials#112]
Thank you very much for integrating different models into the huggingface and creating excellent tutorials on how to use them.

I would like to fine-tune the  [Swin model](https://huggingface.co/docs/transformers/model_doc/swin) on my custom dataset. How to fine-tune the Swim model for the object detection task when the labeled set has only bounding box labels but does not have masking labels? The notebook file showcasing how to fine-tune the Swin model for object detection on a custom dataset will be much more useful. If anyone has done this please let me know. Any type of help is appreciated. Thank you very much!
